### DISCUSSION TITLE
I can not use server components in /pages folder.

### DISCUSSION THREAD

**Discussion by Lrunlin** [vercel/next.js#44895] [Help]
I want to use "use" in the components of the pages folder to request data, but when I import use in reactJS, it is undefined
![image](https://user-images.githubusercontent.com/54397964/212543450-0ce7eac6-87f9-4f34-a6c2-74d5130bfd27.png)
![image](https://user-images.githubusercontent.com/54397964/212543460-d8cc0e81-46fd-4d2f-bb06-6d36b54ab27f.png)

**Comment by Lrunlin** [comment]
I didn't open appDir in config.js. I just want to use/pages+server component

**Comment by icyJoseph** [comment]
Server components are specifically scoped to appDir.

**Reply by Lrunlin** [reply]
Thank you for your answer. I'll try
